browser/js/admin/admin.user/admin.user.controller.js

```diff
@@ -0,0 +1,25 @@
+app.controller('AdminUserCtrl', function($scope, $state, getAllUsers, UserFactory) {
+
+  $scope.users = getAllUsers;
```

Comment:
This should probably be a getter function that retrieves a cache from the factory